.circleci/config.yml

```diff
@@ -422,3 +438,4 @@ workflows:
     jobs:
       - deb-tests
       - translation-tests
+      - docs-linkcheck
```

Comment:
Note to other reviewers: `docs-linkcheck` cron settings are unrelated to the kernel updates, but sure are nice to have. :slightly_smiling_face: